apidoc/Titanium/UI/WebView.yml

```diff
@@ -1027,6 +1027,13 @@ properties:
     since: "1.8.0"
     default: true
 
+  - name: fill
```

Comment:
I am not a fan of another platform-specific property here. How does iOS behave? If it's the same, we should keep it the same, if it's different, they should be aligned.

Reply:
Good point. I didn't check iOS as the issue was only reported on Android. I can check that later. If it will show a blue background I'll remove the flag and make it the default

Reply:
yes, iOS is blue:

![Simulator Screenshot - iPhone 16 Plus - 2024-11-09 at 19 50 09](https://github.com/user-attachments/assets/f12bab46-a14b-4750-b1b3-a93ce4512561)

so I've made it a default value for Android :+1: